### ISSUE TITLE
With .NET 6 SDK, the default tfm is net5.0 not net6.0 for created project.

### ISSUE THREAD

**Issue by ryalanms** [dotnet/wpf#3964]
The project templates are located here: 

https://github.com/dotnet/wpf/tree/master/packaging/Microsoft.Dotnet.Wpf.ProjectTemplates

**Comment by thomasclaudiushuber** [comment]
I can pick this one.

@ryalanms I saw there are also some ".NET Core" things in the descriptions, should this also be changed to just .NET with this issue?

For example here: https://github.com/dotnet/wpf/blob/master/packaging/Microsoft.Dotnet.Wpf.ProjectTemplates/Microsoft.DotNet.Wpf.ProjectTemplates.ArchNeutral.csproj

It says
```
<PackageDescription>Project templates for .NET Core WPF Projects</PackageDescription>
```

I would drop the "Core" here from the text. Also at other places.
```
<PackageDescription>Project templates for .NET WPF Projects</PackageDescription>
```

**Comment by ryalanms** [comment]
Hi @thomasclaudiushuber.  Yes, .NET Core can be changed to .NET.  Thanks.

**Comment by ryalanms** [comment]
Fixed by @thomasclaudiushuber: #3976.  Closing...
